src/app/components/HoldRequest/HoldRequest.jsx

```diff
@@ -314,8 +314,9 @@ class HoldRequest extends React.Component {
     const itemSource = selectedItem.itemSource;
     const deliveryLocations = this.props.deliveryLocations;
     const isEddRequestable = this.props.isEddRequestable;
+    const allClosed = closedLocations.includes('');
```

Comment:
This is definitely nicer and more readable, but as this is used in a couple of different jsx files, would this be better as a shared function / method?